www/js/insert-slide.js

```diff
@@ -4,6 +4,11 @@ const electron = require('electron');
 const analytics = electron.remote.getGlobal('analytics');
 const tingle = require('./vendor/tingle');
 const strings = require('./strings');
+const settings = require('./settings');
+
+const { store } = electron.remote.require('./app');
+
+let announcementOverlay = store.getUserPref('app.announcement-overlay');
```

Comment:
something buggy with this. It only seems to retrieve the defaults value. The checkbox is checked everytime the modal is opened regardless of the value in the setting. ideally they should be in sync.